uint32 frequency
---
bool result
